src/main/java/io/stargate/sgv2/jsonapi/service/resolver/ListTablesCommandResolver.java

```diff
@@ -0,0 +1,39 @@
+package io.stargate.sgv2.jsonapi.service.resolver;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.stargate.sgv2.jsonapi.api.model.command.CommandContext;
+import io.stargate.sgv2.jsonapi.api.model.command.impl.ListTablesCommand;
+import io.stargate.sgv2.jsonapi.service.cqldriver.CQLSessionCache;
+import io.stargate.sgv2.jsonapi.service.cqldriver.executor.KeyspaceSchemaObject;
+import io.stargate.sgv2.jsonapi.service.operation.Operation;
+import io.stargate.sgv2.jsonapi.service.operation.tables.ListTablesOperation;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+/** Command resolver for the {@link ListTablesCommand}. */
+@ApplicationScoped
+public class ListTablesCommandResolver implements CommandResolver<ListTablesCommand> {
+  private final ObjectMapper objectMapper;
+  private final CQLSessionCache cqlSessionCache;
+
+  @Inject
+  public ListTablesCommandResolver(ObjectMapper objectMapper, CQLSessionCache cqlSessionCache) {
+    this.objectMapper = objectMapper;
+    this.cqlSessionCache = cqlSessionCache;
+  }
+
+  /** {@inheritDoc} */
+  @Override
+  public Class<ListTablesCommand> getCommandClass() {
+    return ListTablesCommand.class;
+  }
+
+  /** {@inheritDoc} */
+  @Override
+  public Operation resolveKeyspaceCommand(
+      CommandContext<KeyspaceSchemaObject> ctx, ListTablesCommand command) {
```

Comment:
This should be using the OperationAttempt model

Reply:
We didn't have a cql command executed in here, so did not use the OperationAttempt model.